src/api/routes/quote.ts

```diff
@@ -0,0 +1,79 @@
+import { isObjectEmpty } from './../../utils/common';
+import express from 'express';
+import { IQuote } from './../../models/quoteModel';
+import { quoteManager } from '../../managers';
+import auth from '../middlewares/auth';
+import { Errors } from '../../utils/errors';
+
+const router = express.Router();
+
+/**
+ * Get /api/quote
+ * Public
+ * Get all berakhot
+ */
+router.get('/', async function(req, res) {
+  const quotes = await quoteManager.getQuotes();
+  res.status(200).send(quotes);
+});
+
+/**
+ * GET /api/quote/random
+ * Public
+ * Get random quote
+ */
+router.get('/random', async function(req, res) {
+  console.log('test');
+  const quote = await quoteManager.getRandomQuote();
+  res.status(200).send(quote);
+});
+
+/**
+ * GET /api/quote/:id
+ * Public
+ * Get quote by id
+ */
+router.get('/:id', async function(req, res) {
+  const quote = await quoteManager.getQuoteById(req.params.id);
+  res.status(200).send(quote);
+});
+
+/**
+ * POST /api/quote
+ * Private
+ * Add quote
+ */
+router.post('/', auth, async function(req, res) {
+  const { title, writer }: IQuote = req.body;
+
+  // Simple validations
+  if (!title) return res.status(400).json({ msg: Errors.TitleRequired });
```

Comment:
In the future, we can refactor simple validation shit to middleware / use third party packages.

But for now, it's good!

Reply:
Actually I don't think we need this validation, because mongoose doing it for us, isn't he?

Reply:
I removed this simple validation